### ISSUE TITLE
Tick Blue

### ISSUE THREAD

**Issue by cassianogo** [Rhymen/go-whatsapp#14]
How send message `["Presence",{"id":"1234567890@c.us","type":"available"}]` in my code??  For the user to see  that I read the message.

**Comment by Rhymen** [comment]
I think the presence message is used to display your current online status and not to mark the message as read. You can send the presence request with ```wac.Acknowledge(jid)```, but the other request is not implemented yet.

**Comment by cassianogo** [comment]
How do I send the message presence request with `wac.Acknowledge(jid)`? In `HandleTextMessage` ?

**Comment by Rhymen** [comment]
That depends on what you want to do. I'm not entirely sure how to help you with that, but if you share your code I can take a look at it.

**Comment by Rhymen** [comment]
I renamed the `Acknowledge` function to `Presence` and added a `presence` parameter. The parameter can be one of `whatsapp.PresenceAvailable`, `whatsapp.PresenceUnavailable` and `whatsapp.PresenceComposing`. Sadly I cannot check if this is working or not, because I currently only have one phone on me.
I added a `wac.Read(jid, id string)` function to set the two blue ticks as you requested. If you call this function inside of your TextMessageHandler like so `wac.Read(message.Info.RemoteJid, message.Info.Id)` it should mark every incoming message as read.
